### DISCUSSION TITLE
4.6.13

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#296] [Announcements]
#### FIXED
- 修复无损音质获取失败时视频无法播放
- 修复使用寒霜弹幕使播放6分钟后无弹幕
- 修复视频动态页视频时长显示错误
- 修复下载视频切换剧集时弹窗报错
- 修复设置画面比例后画面大小不随窗口大小变化
- 修复直播间醒目留言无法显示发送者名字
  - 感谢 @GD-Slime https://github.com/ywmoyue/biliuwp-lite/pull/291
- 修复部分up主的视频详情页无推荐视频和无法分享
  - 感谢 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/292

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.13'>4.6.13</a>.</em>

**Comment by jnbadnsodkns** [comment]
自4.6.12开始疯狂闪退，至今无解

**Reply by ywmoyue** [reply]
@fengranmoxia 
请尝试卸载应用后使用安装教程中的脚本安装方式进行安装然后看是否还会闪退

如果还会闪退，请将日志文件通过邮箱发送给我，日志文件在应用设置帮助页中能找到，我的邮箱地址在应用设置关于页中
